docs/security-model.mdx

```diff
@@ -3,69 +3,89 @@ id: security-model
 title: Security Model
 ---
 
-The security model used by Ory is designed to deliver the most secure and hassle-free solution to all users.
+Orys' security model delivers the most secure and hassle-free solution to users
+by managing `sessions`, `tokens` and `cookies`.
 
-To achieve that, Ory manages:
+Although it is an uncommon approach that might require a shift in thinking, it's
+implemented so that you don't have to worry about refreshing tokens or deciding
+whether to store them in `localStorage` or `document.cookies`.
 
-- sessions
-- tokens
-- cookies
```

Comment:
Oh, what happened here? The list was there to bring attention to the three elements Ory manages. 
Also, these weren't put into `code` bc. in this context it doesn't make sense IMO - these are not used as parameter names or code excerpts, but rather as proper names of elements of some system. Could you revert to the way it was before? 

Also, what's up with the line break? Lines look way shorter than the 130 char. limit we have defined.

Reply:
@Benehiko Well it's just an introductory sentence, we're explaining something, giving ppl an idea of what they can expect. Such general statements are not perfect, sure, but they're not the devil ;) So as long as they're not misleading I don't see a problem with using them sparingly.

Not to point fingers, but the `heavy lifting` sentence you wanted to use is just as verbose as this one haha :D